client/src/ViewTendersButton.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const ViewTendersButton = ({ onClick }) => (
```

Comment:
Why did you create a custom component for this? It seems like it doesn't add anything to a plan button

Reply:
I have removed and added a plain button

Reply:
You need to delete this file

Reply:
done